book/chapters/09-boot/contents.tex

```diff
@@ -686,8 +709,8 @@ \section{Pornirea kernelului și drivere}
 și apoi pornește procesul \texttt{init}, primul proces al sistemului, care, la rândul său,
 va porni alte procese. Un driver este un fișier ce conține codul
 pentru utilizarea și configurarea unui dispozitiv hardware. După investigarea
-hardware-ului sistemului, nucleul sistemului de operare identifică dispozitivele
-hardware și apoi încarcă de pe disc device driverele.
+hardware-ului sistemului, nucleul sistemului de operare va anunta un program din user-space
```

Comment:
Nu, nu zice de udev. Lasă așa, mai puțin precis, cu "identifică dispozitivele hardware"

Reply:
dar... kernelul nici macar nu poate sa incarce module. Ori apeleaza modprobe inainte sa porneasca init. Ori apeleaza udev, sau programe similare.